packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -17,24 +17,36 @@ export enum OrgType {
 export async function getWorkspaceOrgType(
   defaultUsernameOrAlias?: string
 ): Promise<OrgType> {
-  if (isNullOrUndefined(defaultUsernameOrAlias)) {
+  if (defaultUsernameOrAlias === null || defaultUsernameOrAlias === undefined) {
```

Comment:
I changed this b/c I was getting a warning that isNullOrUndefined() is deprecated